examples/cors/backend/Dockerfile-service

```diff
@@ -6,4 +6,4 @@ RUN mkdir /code
 ADD ./service.py /code/
 ADD ./start_service.sh /usr/local/bin/start_service.sh
 RUN chmod u+x /usr/local/bin/start_service.sh
-ENTRYPOINT /usr/local/bin/start_service.sh
+ENTRYPOINT ["/bin/bash", "/usr/local/bin/start_service.sh"] 
```

Comment:
Is this explicit "/bin/bash" required?

Reply:
Unfortunately it is. It appears that the CRLF obscures the shebang at the top of the script somehow. Without this explicit instruction,  the container immediately exits with error 1 `exec user process caused "no such file or directory"`, as opposed to error 127 when the ENTRYPOINT is in non-exec format.

I know it's ugly, but it works, and at no meaningful cost.

As an alternative, I could also rewrite the Dockerfiles and move the contents of the shell scripts into them. The scripts are only a couple lines long, and I don't really understand why they're there. A Dockerfile is ultimately a shell script wrapped in a bunch of additional features. It makes more sense to me if they actually contain the shell instructions they run, instead of containing references to their instructions.

Reply:
i think moving the scripts into the Dockerfiles would simplify things and make recipe more explicit. It would also remove the need to `ADD` and chmod these.

if you do go  with execing i would consider using `/bin/sh` rather than bash as its more portable